package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "perimeterx-node-core",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
since we're adding configurations here, this should be a minor version, not a patch